pkg/kubectl/cmd/util/printing.go

```diff
@@ -30,6 +33,32 @@ func AddPrinterFlags(cmd *cobra.Command) {
 	cmd.Flags().StringP("template", "t", "", "Template string or path to template file to use when -o=template or -o=templatefile.  The template format is golang templates [http://golang.org/pkg/text/template/#pkg-overview]")
 }
 
+// AddOutputFlags adds output related flags to a command. Used by mutations only.
+func AddOutputFlags(cmd *cobra.Command) {
+	cmd.Flags().StringP("output", "o", "", "Output mode. Use -o name for showing only kind/name.")
+}
+
+// PrintSuccess prints message after finishing mutating operations
+func PrintSuccess(cmd *cobra.Command, out io.Writer, resource string, name string, operation string) {
+	outputMode := GetFlagString(cmd, "output")
```

Comment:
Just make the output mode a parameter to the function so that this function doesn't need the whole command.